src/codeCache.h

```diff
@@ -29,6 +29,12 @@ enum ImportId {
     NUM_IMPORTS
 };
 
+enum ImportType {
+    PRIMARY,
+    SECONDARY,
+    NUM_IMPORT_TYPES,
```

Comment:
Please remove `,` after `NUM_IMPORT_TYPES` as it will always be the last constant